packages/integrations/mdx/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@astrojs/mdx",
+  "description": "Use MDX within Astro",
+  "version": "0.0.1",
+  "type": "module",
+  "types": "./dist/index.d.ts",
+  "author": "withastro",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/withastro/astro.git",
+    "directory": "packages/integrations/mdx"
+  },
+  "keywords": [
+    "astro-component",
+    "renderer",
+    "mdx"
+  ],
+  "bugs": "https://github.com/withastro/astro/issues",
+  "homepage": "https://astro.build",
+  "exports": {
+    ".": "./dist/index.js",
+    "./client.js": "./client.js",
+    "./server.js": "./server.js",
```

Comment:
The files `client.js` and `server.js` don't seem to be contained in the package. If you're not planning to add them, we should maybe remove them from here.

Reply:
Nice catch!